src/navigator/Screens.tsx

```diff
@@ -86,6 +86,7 @@ export enum Screens {
   Support = 'Support',
   SupportContact = 'SupportContact',
   SwapScreen = 'SwapScreen',
+  SwapScreenWithBack = 'SwapScreenWithBack',
```

Comment:
Another option for the name is just `Swap`, we don't include the `Screen` suffix for many screens.